tools/c7n_azure/c7n_azure/resources/key_vault_keys.py

```diff
@@ -28,7 +28,7 @@
 log = logging.getLogger('custodian.azure.keyvault.keys')
 
 
-@resources.register('keyvault-keys')
+@resources.register('keyvault-keys', aliases=['keyvault-key'])

```

Comment:
I'd update primary name & move `keyvault-keys` to alias.. I'd treat alias as something that can be deprecated.

We need to update docs too.